frontend/app_flowy/packages/appflowy_editor/lib/src/service/shortcut_event/built_in_shortcut_events.dart

```diff
@@ -159,6 +160,13 @@ List<ShortcutEvent> builtInShortcutEvents = [
     linuxCommand: 'ctrl+u',
     handler: formatUnderlineEventHandler,
   ),
+  ShortcutEvent(
+    key: 'Toggle Checkbox',
```

Comment:
`CTRL + Q`? Normally, `CTRL + Q` means quitting the app. 

I suggest using `CTRL + ENTER` or `META + ENTER` instead.